tests/unit/TodoList2.spec.ts

```diff
@@ -57,6 +61,95 @@ describe('TodoList - common expansion panel header tests', () => {
     wrapper.destroy()
   })
 
+  it('display affiliation invitation todo (request access, open)', async () => {
+    rootStore.tasks = []
+    businessStore.businessInfo = Object.assign(businessStore.businessInfo, { identifier: 'testIdentifier' })
+
+    const sandbox = sinon.createSandbox()
+    const fetchAffiliationInvites = sandbox.stub(AuthServices, 'fetchAffiliationInvitations')
+    // feature flag override
+    const featureFlag = sandbox.stub(utils, 'GetFeatureFlag')
+
+    featureFlag.withArgs('enable-affiliation-invitation-request-access')
+      .returns(true)
+
+    fetchAffiliationInvites.returns(new Promise(
+      resolve => resolve(
+        {
+          config: undefined,
+          headers: undefined,
+          status: 200,
+          statusText: '',
+          data: {
+            affiliationInvitations: [
+              // returns 3 items, but only 1st one should be displayed
+              {
+                id: 12,
+                type: 'RequestAccess',
+                status: 'ACTIVE',
+                business: {
+                  businessIdentifier: 'BC0871427',
+                  name: '0871427 B.C. LTD.',
+                  corpType: { code: 'BC' }
+                },
+                toOrg: { name: 'Two Monkeys are friends Corp.', id: 3113 },
+                fromOrg: { name: 'Tree Frog Design Inc.', id: 1114 }
+              },
+              {
+                id: 17,
+                type: 'RequestAccess',
+                status: 'DELETED',
+                business: {
+                  businessIdentifier: 'BC0871427',
+                  name: '0871427 B.C. LTD.',
+                  corpType: { code: 'BC' }
+                },
+                toOrg: { name: 'Two Monkeys are friends Corp.', id: 3113 },
+                fromOrg: { name: 'Some other org', id: 8787 }
+              },
+              {
+                id: 19,
+                type: 'MagicLink',
+                status: 'ACTIVE',
+                business: {
+                  businessIdentifier: 'BC0871427',
+                  name: '0871427 B.C. LTD.',
+                  corpType: { code: 'BC' }
+                },
+                toOrg: { name: 'Two Monkeys are friends Corp.', id: 3113 },
+                fromOrg: { name: 'Some third org', id: 8787 }
+              }
+            ]
+          }
+        }
+      )))
+
+    const wrapper = mount(TodoList, { vuetify })
+    await Vue.nextTick()
+    await flushPromises()
+
+    expect(wrapper.findAll('.todo-item').length).toEqual(1)
+    // verify title starts with
+    expect(wrapper.find('.list-item__title').text())
+      .toContain('Request for authorization to mange this business')
+
+    // verify subtitle has information
+    expect(wrapper.find('.todo-subtitle').text()).toBe('From: Tree Frog Design Inc.')
+
+    // verify both buttons authorize and do not authorize are visible
+    const actionBtns = wrapper.findAll('.width-45-float-right')
```

Comment:
It would be better to assign a classname to your buttons that refers to its purpose, instead of this styling classname. I think you can just rename the existing class -- it will serve as both identifying the type of button and setting some CSS for it.

(I'm trying to avoid confusion between styling classes and "functional" classes. Styling classes should be used for styling only. Functional classes can help identify what the button is for and also include styling/CSS if needed. Another way to look at it is that "all affiliation buttons have width 45% and float-right, but not all width-45-float-right buttons are necessarily affiliation buttons".)